### DISCUSSION TITLE
Can share with windows client?

### DISCUSSION THREAD

**Discussion by cmaggic22** [dorssel/usbipd-win#730] [Q&A]
I Want to share a security dongle between 2 windows machines, is that possible ? (I see that all examples share whith WSL client)
I've installed the usbipd on the machine having the dongle with ip 192.168.0.80
To start sharing I did: usbipd bind --busid=2-2

Then installled the usbipd in the client windows machine that wants to connect to the shared dongle 
But when I try to attach to the other machine using
usbipd attach --remote=192.168.0.80 --busid=2-2
got error:
Unrecognized command or argument 'attach'
Regards

**Comment by dorssel** [comment]
A quick search through older discussions and issues:

https://github.com/dorssel/usbipd-win/discussions?discussions_q=is%3Aopen+windows+client

https://github.com/dorssel/usbipd-win/issues/65

TL;DR: `usbipd-win` is server side only. Other users have had some success with Windows clients (see links), but that's outside the scope of this project.
